common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -1,15 +1,15 @@
 import { TFetchCCRates } from 'actions/rates';
 import { Identicon } from 'components/ui';
 import { NetworkConfig } from 'config/data';
-import { Ether } from 'libs/units';
-import { IWallet } from 'libs/wallet';
+import { IWallet, Balance } from 'libs/wallet';
 import React from 'react';
 import translate from 'translations';
 import { formatNumber } from 'utils/formatters';
+import Spinner from 'components/ui/Spinner';
 import './AccountInfo.scss';
 
 interface Props {
-  balance: Ether;
+  balance: Balance;
```

Comment:
`balance` is still `Ether` in BalanceSidebar/index.tsx, and potentially other places. Check the CI logs or run `npm run tslint` to get the full log.